src/media/playback/playback_sensor.h

```diff
@@ -79,7 +79,8 @@ namespace librealsense
 
         rsutils::subscription register_options_changed_callback( options_watcher::callback && cb ) override
         {
-            throw not_implemented_exception( "Registering options value changed callback is not implemented for this sensor" );
+            LOG_ERROR( "Registering options value changed callback is not implemented for this sensor" );
```

Comment:
Why? I don't think this is good...?

Reply:
If we throw then the viewer cannot play recorded files

Reply:
Reverted change. Viewer handling the exception